src/couch_server.erl

```diff
@@ -555,7 +555,12 @@ delete_file(RootDir, FullFilePath, Options) ->
 rename_on_delete(Original) ->
     DeletedFileName = deleted_filename(Original),
     case file:rename(Original, DeletedFileName) of
-        ok -> {ok, {renamed, DeletedFileName}};
+        ok ->
```

Comment:
parts of this change seem to be in the first commit (a file:change_time call is in there)